nanoFramework.System.Net.Http/Http/System.Uri.cs

```diff
@@ -261,14 +297,19 @@ public string AbsolutePath
                     throw new InvalidOperationException();
                 }
 
-                return _AbsolutePath;
+                if(_absolutePath == null)
```

Comment:
why not initialize _absolutePath with string.Empty? So this won't be necessary as it will always have a value?

Reply:
I was following the existing pattern of using null values and checking for null later.  However upon inspection, this has no real benefit so I removed it in the commit i'm about to submit.